inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -593,6 +596,7 @@ std::shared_ptr<CNNNetworkImpl> convertFunctionToICNNNetwork(const std::shared_p
                 std::make_shared<Builder::NodeConverter<::ngraph::op::TileIE>>(),
                 std::make_shared<Builder::NodeConverter<::ngraph::op::v1::TopK>>(),
                 std::make_shared<Builder::NodeConverter<::ngraph::op::TopKIE>>(),
+                // std::make_shared<Builder::NodeConverter<::ngraph::op::Transpose>>(),
```

Comment:
why is commented? 

CC @GlebKazantaev

Reply:
Build error in KMB-plung.

Reply:
openvino/bin/intel64/Debug/lib/libinference_engine_legacy.so: undefined reference to `InferenceEngine::Builder::NodeConverter<ngraph::op::v1::Transpose>::createLayer(std::shared_ptr<ngraph::Node> const&) const'
collect2: error: ld returned 1 exit status
tools/model-comparator/CMakeFiles/model-comparator.dir/build.make:96: recipe for target '/home/tan/projects/openvino/bin/intel64/Debug/model-comparator' failed
make[2]: *** [/home/tan/projects/openvino/bin/intel64/Debug/model-comparator] Error 1
CMakeFiles/Makefile2:1587: recipe for target 'tools/model-comparator/CMakeFiles/model-comparator.dir/all' failed
make[1]: *** [tools/model-comparator/CMakeFiles/model-comparator.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....

Reply:
Transpose was already moved to use visitor api on master.  Looks like that was merge conflict. Just remove this line.

Reply:
Ok. Row is deleted.